src/write/file.rs

```diff
@@ -55,6 +55,21 @@ pub struct FileWriter<W: Write> {
     page_specs: Vec<Vec<Vec<PageWriteSpec>>>,
     /// Used to store the current state for writing the file
     state: State,
+    // when the file is written, metadata becomes available
+    metadata: Option<FileMetaData>,
+}
+
+/// Writes a parquet file containing only the header and footer
+///
+/// This is used to write the metadata as a separate Parquet file, usually when data
+/// is partitioned across multiple files.
+///
+/// Note: Recall that when combining row groups from [`FileMetaData`], the `file_path` on each
+/// of their column chunks must be updated with their path relative to where they are written to.
+pub fn write_metadata_sidecar<W: Write>(writer: &mut W, metadata: &FileMetaData) -> Result<u64> {
+    let mut len = start_file(writer)?;
```

Comment:
Ah good catch! I was able to verify that a `_metadata` file written with pyarrow had `PAR1` as both the first four and the last four bytes.